ambry-commons/src/main/java/com.github.ambry.commons/BlobId.java

```diff
@@ -135,7 +136,11 @@
    * @param accountId The id of the {@link Account} to be embedded into the blob. Only relevant for V2 and above.
    * @param containerId The id of the {@link Container} to be embedded into the blob. Only relevant for V2 and above.
    * @param partitionId The partition where this blob is to be stored. Cannot be {@code null}.
-   * @param isEncrypted {@code true} if blob that this blobId represents is encrypted. {@code false} otherwise
+   * @param isEncrypted {@code true} if blob that this blobId represents is encrypted. {@code false} otherwise.
+   *                                For {@link BlobId#BLOB_ID_V3}, isEncrypted is valid in frontend's memory for
```

Comment:
shouldn't make references to frontend/server etc. Let's chat about how to do this.